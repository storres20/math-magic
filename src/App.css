/* Calculator.js */
.calcContainer {
  display: flex;
  justify-content: flex-end;
}

.calc {
  display: flex;
  flex-direction: column;
  align-items: flex-end;
}

.output {
  display: flex;
  flex-direction: row-reverse;
  align-items: center;
  background-color: #858693;
  color: white;
  height: 3em;
  width: 100%;
}

.horizontalBtn {
  display: flex;
  flex-direction: row;
}

.verticalBtn {
  display: flex;
  flex-direction: column;
}

.btnGray {
  background-color: #e0e0e0;
}

.btnOrange {
  background-color: #f5913e;
}

button {
  border: 0.5px solid #d1d1d1;
  width: 4em;
  height: 3em;
  font-size: larger;
}
