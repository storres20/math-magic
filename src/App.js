import React from 'react';
import { Route, Routes } from 'react-router-dom';
import './App.css';
// import Calculator from './components/Calculator';
import Navbar from './components/Navbar';
import Calc from './pages/Calc';
import Home from './pages/Home';
import Quote from './pages/Quote';

function App() {
  return (
    <div>
      <Navbar />

      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="/calculator" element={<Calc />} />
        <Route path="/quote" element={<Quote />} />
      </Routes>

    </div>
  );
}

export default App;
