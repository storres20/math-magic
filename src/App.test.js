import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import App from './App';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <App />
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
