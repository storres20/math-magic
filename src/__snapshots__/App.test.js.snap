// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="NavContainer"
  >
    <h2>
      Math Magicians
    </h2>
    <ul
      className="Links"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <li>
        |
      </li>
      <a
        className=""
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <li>
        |
      </li>
      <a
        className=""
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </ul>
  </div>
  <div>
    <h2>
      Welcome to our page!
    </h2>
    <p>
      Ut occaecat incididunt fugiat eiusmod est dolor esse. Minim nisi anim commodo exercitation esse culpa voluptate sunt ullamco reprehenderit. Elit mollit aliquip non sit ut minim eu aute Lorem est in id. Enim tempor enim eu mollit eiusmod ad quis ad eu.
    </p>
    <p>
      Et fugiat ipsum sunt sint commodo pariatur ex laboris officia cillum fugiat nisi adipisicing. Reprehenderit id consectetur magna dolore minim dolor adipisicing anim ea ex pariatur sit esse. Aliqua officia culpa voluptate velit. Non in pariatur eu sit sit ex commodo labore laboris occaecat sint et nostrud.
    </p>
  </div>
</div>
`;
